values/roman/prod-values.example.yaml

```diff
@@ -0,0 +1,21 @@
+roman:
+  replicaCount: 1
+  image:
+    repository: quay.io/wire/roman
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: staging
+  serviceToken: # Provide the roman service token.
+  # randomly generated for local testing
+  appKey: b53181dd-6400-4960-8988-f775545588ff-0949f503-421e-4588-a2c5-f64fd9c180fd
+  wireApiHost: https://nginz-https.example.com
```

Comment:
this is totally wrong. look at the prod-values.example.yaml . when referring to a wire component, we don't use FQDNs.